src/parser/deathknight/blood/modules/features/MarrowrendUsage.js

```diff
@@ -196,10 +195,15 @@ class MarrowrendUsage extends Analyzer {
             Avoid casting Marrowrend unless you have {this.refreshAtStacks} or less stacks or if Bone Shield has less than 6sec of its duration left.
           </>
         )}
-      />
+      >
+        <BoringSpellValueText spell={SPELLS.MARROWREND}>
+          <>
+            {this.badMRCasts} / {this.totalMRCasts} <small>Bad Marrowrend casts</small>
```

Comment:
Same, `bad casts`

Reply:
I agree with your comments and I'll make the changes later today.